lib/get-group-tangle.js

```diff
@@ -12,29 +12,31 @@ const { isIdentityGroupSSBURI } = require('ssb-uri2')
 
 const strategy = new Strategy({})
 
-module.exports = function GetGroupTangle(server) {
-  const getUpdates = GetUpdates(server)
+/** `server` is the ssb server you're using. `tangle` is the name of the tangle in the group you're looking for, e.g. "group" or "member" */
+module.exports = function GetGroupTangle(server, tangle) {
+  const getUpdates = GetUpdates(server, tangle)
 
-  return function getGroupTangle(groupId, cb) {
+  return function getTangle(groupId, cb) {
     if (!isIdentityGroupSSBURI(groupId)) {
       // prettier-ignore
       return cb(new Error(`get-group-tangle expects valid groupId, got: ${groupId}`))
     }
 
     server.box2.getGroupKeyInfo(groupId, (err, info) => {
       // prettier-ignore
-      if (err) return cb(clarify(err, 'Failed to get group key info when getting group tangle'))
+      if (err) return cb(clarify(err, 'Failed to get group key info when getting a group tangle'))
 
       if (!info) {
         return cb(new Error(`get-group-tangle: unknown groupId ${groupId}`))
       }
 
       getUpdates(info.root, (err, msgs) => {
+        // prettier-ignore
         if (err) return cb(clarify(err, 'Failed to read updates when getting group tangle'))
 
         const nodes = msgs.map((msg) => ({
           key: msg.key,
-          previous: msg.value.content.tangles.group.previous,
+          previous: msg.value.content.tangles[tangle].previous,
         }))
         // NOTE: getUpdates query does not get root node
         nodes.push({ key: info.root, previous: null })
```

Comment:
yeah see here we don't even look up the root message to check it's tangle, we assume it's just this.
So modifying the group-root to add this info isn't that useful IMO

Reply:
yeeah maybe not, i'm just imagining some poor person in a bunch of years having to poke at broken graphs and being angry